doc/Installation/Installation-CentOS-7-Apache.md

```diff
@@ -118,8 +136,15 @@ Install the policy tool for SELinux:
     semanage fcontext -a -t httpd_sys_content_t '/opt/librenms/bootstrap/cache(/.*)?'
     semanage fcontext -a -t httpd_sys_rw_content_t '/opt/librenms/bootstrap/cache(/.*)?'
     restorecon -RFvv /opt/librenms/bootstrap/cache/
+    semanage fcontext -a -t httpd_sys_content_t '/opt/librenms/cache(/.*)?'
```

Comment:
It might be better to remove usage of this directory...

Reply:
When I do this, creating a new device group fails with a 500 error and librenms.log shows that it can't write to that directory. Do you have any good context suggestion ?

Reply:
Your fix is correct, but the better fix is for LibreNMS to stop using that directory.  That is all I am saying.

Reply:
Sorry, didn't understand :) Maybe we could leave this doc until the directory is not used anymore ?